maplayer.c

```diff
@@ -1604,9 +1604,19 @@ char* msLayerBuildSQLOrderBy(layerObj *layer)
   if( layer->sortBy.nProperties > 0 ) {
     int i;
     for(i=0;i<layer->sortBy.nProperties;i++) {
-      char* escaped = msSmallMalloc(strlen(layer->sortBy.properties[i].item) + 3);
-      sprintf(escaped, "\"%s\"", layer->sortBy.properties[i].item);
-      if( i > 0 )
+      char* escaped = msLayerEscapePropertyName(layer, layer->sortBy.properties[i].item);
+      //Enclose property name in double quotes (if it isn't yet) to ensure that mixed case property names are supported
+      if (escaped[0] != '"')
```

Comment:
tx, so I'll undo my initial changes on the escaping stuff and just de-activate msOracleSpatialEscapePropertyName, that's fine. @msmitherdc you created the function - should I remove it?
With regards to git should I create an undo commit for my initial work on the escaping? I guess we cannot hide the first commit as it is already published in the branch used for the pull request?
Or should I drop the branch and create a new "clean" one?